libcontainer/selinux/selinux.go

```diff
@@ -33,19 +33,29 @@ const (
 )
 
 var (
-	assignRegex           = regexp.MustCompile(`^([^=]+)=(.*)$`)
-	mcsList               = make(map[string]bool)
-	mcsLock               sync.Mutex
-	selinuxfs             = "unknown"
-	selinuxEnabled        = false // Stores whether selinux is currently enabled
-	selinuxEnabledChecked = false // Stores whether selinux enablement has been checked or established yet
+	assignRegex                   = regexp.MustCompile(`^([^=]+)=(.*)$`)
+	mcsList                       = make(map[string]bool)
+	mcsLock                       sync.Mutex
+	selinuxfs                     = "unknown"
+	selinuxEnabled                = false // Stores whether selinux is currently enabled
+	selinuxEnabledHost            = false // Stores whether selinux is currently enabled on the host
+	defProcessLabel, defFileLabel = getDefaultLabels()
 )
 
 type SELinuxContext map[string]string
 
+func init() {
```

Comment:
Actually this not not good to do in a package level init.  It slows down binary startup time and other issues with this always being run when you import the package.  

This should be done lazily with a sync.Once or some other way.  We don't want to make out of process calls in init functions in Go.